src/cordova/apple/ios.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,17 +5,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/CocoaLumberjack/CocoaLumberjack",
       "state" : {
-        "revision" : "80ada1f753b0d53d9b57c465936a7c4169375002",
-        "version" : "3.7.4"
+        "revision" : "0188d31089b5881a269e01777be74c7316924346",
```

Comment:
Let's revert this for now. No need to upgrade the dependencies. It's already an intrusive change.